common/src/main/java/dev/architectury/event/EventFactory.java

```diff
@@ -63,28 +61,6 @@ protected Object handleInvocation(@NotNull Object proxy, @NotNull Method method,
         }));
     }
     
-    @SafeVarargs
-    public static <T> Event<T> createInteractionResult(T... typeGetter) {
```

Comment:
We should mark these as `@Deprecated` in 1.x to let people creating their own events know they are being removed in favour of `EventResult`s